x-pack/plugins/security_solution/public/detections/pages/alerts/index.tsx

```diff
@@ -31,7 +31,7 @@ const AlertsContainerComponent: React.FC = () => {
     <Switch>
       <Route path={ALERTS_PATH} exact component={AlertsRoute} />
       {/* Redirect to the alerts page filtered for the given alert id */}
-      <Route path={`${ALERTS_PATH}/:alertId`} component={AlertDetailsRedirect} />
+      <Route path={`${ALERTS_PATH}/redirect/:alertId`} component={AlertDetailsRedirect} />
```

Comment:
Added the `redirect` subpath, to be explicit about the path functionality, but also preserve any future paths we may want to use such as `/alert` or `/:id`, etc...